G_channels.md

```diff
@@ -103,22 +100,31 @@ end
 
 For our chat app, we'll allow anyone to join the `"rooms:lobby"` topic, but any other room will be considered private and special authorization, say from a database, will be required. We won't worry about private chat rooms for this exercise, but feel free to explore after we finish. To authorize the socket to join a topic, we return `{:ok, socket}` or `{:ok, reply, socket}`. To deny access, we return `{:error, reply}`. More information about authorization with tokens can be found in the [`Phoenix.Token` documentation](http://hexdocs.pm/phoenix/Phoenix.Token.html).
 
-With our channel in place, let’s head over to `web/static/js/app.js` and get the client and server talking.
+With our channel in place, let's get the client and server talking. There's some code in `web/static/js/socket.js` to connect to our socket and join our channel already, we just need to set our room name to "rooms:lobby".
 
 ```javascript
-import {Socket} from "deps/phoenix/web/static/js/phoenix"
+...
+socket.connect({token: window.userToken})
```

Comment:
passing params to connect is deprecated. They should now be passed as a `params:` argument to the Socket constructor